src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
```suggestion
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <meta name="description" content="This is an example of a meta description. This will often show up in search results.">
```
Add a meta description tag to up your SEO score.